test/unit/participation.test.ts

```diff
@@ -0,0 +1,97 @@
+import widgets from '../../src/widgets'
+import { JSDOM } from 'jsdom'
+import * as participation from '../../src/participation'
```

Comment:
Why do you pull in the participation module like this? Why not import the methods directly, e.g. `import { renderPartipants, participationObject, recordParticipation, manageParticipation } from '../../src/participation'`?

Reply:
Also, if you do need to do this (which you might want to do because you want to test against the public API, like I do in [tabs test](https://github.com/solid/solid-ui/pull/247/files#diff-2d70df5dd69be0ab433b268762ec8b97R27)), you should _not_ need to do `(participation as any)` - we've created those APIs, not reason to throw it to any type).

Reply:
hmm.. I will look into this.  I didn't understand the difference and just saw that in another place it was done this way. I will change this and try to understand the difference. Meaning the "why" to use it.